museum_export/clean_up_content.py

```diff
@@ -0,0 +1,88 @@
+# clean_up_content.py
+from datetime import datetime
+
+
+class CleanUpContent():
+    """ This class accepts a JSON object representing one item of museum content and massages that to fit our needs for processing. """
+
+    def clean_up_content(self, object):
+        """ This calls all other modules locally """
+        self._define_worktype(object)
+        if 'modifiedDate' in object:
+            object['modifiedDate'] = datetime.strptime(object['modifiedDate'], '%m/%d/%Y %H:%M:%S').isoformat() + 'Z'
+        self._fix_creators(object)
+        self._remove_bad_subjects(object)
+
+    def _fix_creators(self, object):
+        """ Add logic to create a display node for each creator that differs based upon person or corporate contributor. """
+        if "creators" in object:
+            for creator in object["creators"]:
+                life_flag = creator.get("lifeFlag", "")
+                if life_flag == "1":
+                    creator_display = self._get_person_creator_display(creator)
+                elif life_flag == "0":
+                    creator_display = self._get_corporate_creator_display(creator)
+                else:
+                    creator_display = creator.get("fullName", "")
+                creator["display"] = creator_display
+                return
+        return
+
+    def _get_person_creator_display(self, creator):
```

Comment:
I think we should switch this to use my class. 
https://github.com/ndlib/marble-manifest-pipeline/blob/master/process_manifest/creatorField.py

Because this class is unit tested.
https://github.com/ndlib/marble-manifest-pipeline/blob/master/test/test_creatorField.py

I think the class and test should be moved to pipelineutilities.